README.md

```diff
@@ -1,80 +1,14 @@
+This starter guide contains three sections. In the first two sections you will use the "vanilla" Open Telemetry API. In the third section, we will explore features defined in this package and use these tools to better instrument your application. 
 
-The third tutorial will walk you through the annotations and configurations defined in the opentelemetry-contrib-spring package. This tutorial will equip you with new tools to streamline the step up and configuration of OpenTelemetry on Spring and SpringBoot applications.
+The first section will walk you through span creation and propagation using the JavaSDK and the HTTP web client, RestTemplate. 
 
-To use the tools included in this package include the dependency below in your spring application:
+The second section will build on the first. It will walk you through implementing spring-web handler and interceptor interfaces to create traces with minimal changes to your application code. 
 
-
-#### Maven
-```xml
- <dependency>
-    <groupId>io.opentelemetry</groupId>
-    <artifactId>opentelemetry-contrib-spring</artifactId>
-    <version>VERSION</version>
- </dependency>
-```
-
-#### Gradle 
-```gradle
-compile "io.opentelemetry:opentelemetry-contrib-spring:VERSION"
-```
-
-## Features (IN PROGRESS)
-
-Examples and detailed explanations of the features below can be found in tutorial 3. 
-
-### @ConfigTracer
-
-This annotation initializes a global tracer for your application. It will create a Tracer Object which can be injected as a dependency.
-
-@ConfigTracer fields:
-- String: tracerName
- 
-### @TraceMethod  
-
-This annotates a method definition. It wraps a method in a span or logs a method call as an event.
-
-@TraceMethod fields: 
-- String: name
-- Boolean: isEvent 
-
-
-### @TraceClass 
-
-This annotates a class definition. It wraps all public methods in a span or logs all method calls as an event.
-
-@TraceClass fields: 
-- String: name
-- Boolean: isEvent
-- Boolean: includeMethodName (logs method signature)
-
-
-### @TraceRestControllers
-
-This annotates a class definition with @RestController annotation. It creates a new span for rest controllers when a request is received. 
-
-No @TraceRestControllers fields: 
-- Default span name: controllerClassName
-
-
-### @InjectTraceRestTemplate
-
-Inject span context to all requests using RestTemplate.
-
-@InjectTraceRestTemplate fields: 
-- Boolean: isEventLogged (default true) 
-
-### @TraceHibernateDatabase
-
-This annotates a hibernate entity. It wraps all database calls using the Hibernate framework in a span or logs it as an event.
-
-@TraceHibernateDatabase fields: 
-- String: name
-- Boolean: isEvent 
+The third section will walk you through the annotations and configurations defined in the opentelemetry-contrib-spring package. This section will equip you with new tools to streamline the step up and configuration of OpenTelemetry on Spring and Spring Boot applications.
```

Comment:
[Question] If this is the only section that has your contribution, don't you do more than configure, but rather and more importantly also provide means to trace/instrument with minimal changes?

Reply:
True, it handles some configuration but the end goal is instrumentation. I'll make this more explicit